evcache-client/src/main/java/com/netflix/evcache/EVCacheModule.java

```diff
@@ -21,6 +35,8 @@ public EVCacheModule() {
 
     @Override
     protected void configure() {
+    	install(MultibindingsScanner.asModule());
```

Comment:
nit.  You seem to be using 3 space indentation in a project that uses 4 space indentation.  It's best to continue using the same style.

Reply:
fixed.